### ISSUE TITLE
Netflix doesn't work in Safari

### ISSUE THREAD

**Issue by Abrahams95** [daliansky/XiaoMi-Pro-Hackintosh#237]
I can't play any Netflix video on Safari. It gives me error code: S7363-1260-FFFFD089

**Comment by Myrenic** [comment]
I have this problem as well.

**Comment by Menchen** [comment]
https://help.netflix.com/en/node/58823

Seem browser cache issue.

**Comment by Myrenic** [comment]
Tried it, the error is still there.

**Comment by Abrahams95** [comment]
Yeah but on chrome and other browsers you can’t get 1080p. Only Safari has Netflix’s Ultra HD support.

**Comment by yipingw** [comment]
Have you try to use Macintosh?

**Comment by Abrahams95** [comment]
Yes on my old MacBook Pro 15" early 2008 it does work without problems.

**Comment by stevezhengshiqi** [comment]
According to https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.Shiki.en.md#general-faq  

> Can I play HTML5 Netflix videos with Shiki?
No, you cannot since Netflix limited their 1080p content for very few mac models even excluding most Mac Pros.

so our hackintoshes may not support Netflix.
